docs/src/main/asciidoc/neo4j-guide.adoc

```diff
@@ -472,3 +472,61 @@ public class ReactiveFruitResource {
 It exposes its API based on http://www.reactive-streams.org[Reactive Streams], most prominently, as `org.reactivestreams.Publisher`.
 Those can be used directly, but we found it easier and more expressive to wrap them in reactive types from https://projectreactor.io[Project Reactor].
 
+== Configuration Reference
+
+|===
+|Configuration key|Java type|Default Value|Description
+
+|quarkus.neo4j.uri
+|String
+|`bolt://localhost:7687`
+|The uri this driver should connect to. The driver supports `bolt`, `bolt+routing` or `neo4j` as schemes.
+
+|quarkus.neo4j.authentication.username
+|String
+|`neo4j`
+|The login of the user connecting to the database.
+
+|quarkus.neo4j.authentication.password
+|String
+|`neo4j`
+|The password of the user connecting to the database.
+
+|quarkus.neo4j.authentication.disabled
+|boolean
+|false
+|Set this to `true` to disable authentication.
+
+|quarkus.neo4j.pool.metrics-enabled
+|boolean
+|`false`
+|Flag, if metrics are enabled.
+
+|quarkus.neo4j.pool.log-leaked-sessions
+|boolean
+|`false`
+|Flag, if leaked sessions logging is enabled.
+
+|quarkus.neo4j.pool.max-connection-pool-size;
+|int
+|100
+|The maximum amount of connections in the connection pool towards a single database.
+
+|quarkus.neo4j.pool.max-connection-lifetime
+|java.time.Duration
+|`PT1H`
+|Pooled connections older than this threshold will be closed and removed from the pool.
+
+|quarkus.neo4j.pool.connection-acquisition-timeout
+|java.time.Duration
+|`PT1M`
+|Acquisition of new connections will be attempted for at most configured timeout.
+
+|quarkus.neo4j.pool.idle-time-before-connection-test
+|java.time.Duration
+|`PT-0.001S`
```

Comment:
Couldn't you simply use `-1`? Our formatter should support it if it doesn't already and translate that to -1 second.

That being said, I'm not sure it's a safe default value.

Reply:
It's the same default value the driver uses internally, so that is safe. 
I cannot use `-1` because that is a second and not a millisecond and I prefer having the same defaults as the pure driver.